.openshift-ci/runners.py

```diff
@@ -122,6 +126,9 @@ def log_event(self, msg, test_set=None):
         print(f"{marker} {time}: {msg}")
         print(marker)
 
+    def set_provisioned_state(self):
+        subprocess.check_call("tests/e2e/lib.sh set_provisioned_state", shell=True)
```

Comment:
Not going to block on this nit but should we use `common.popen_graceful_kill` here?

Reply:
I don't think it is appropriate in this case i.e. just touching a state file